README.md

```diff
@@ -50,11 +50,12 @@ go test ./pkg/git -run TestCopyServiceWithFailureCopying
 
 This section is temporary. To create a sample promotion Pull Request, until https://github.com/rhd-gitops-example/services/issues/8 is done:
 
-- Copy https://github.com/rhd-gitops-example/gitops-example-dev
-- Copy https://github.com/rhd-gitops-example/gitops-example-staging
-- Build the code: `go build ./cmd/services`
+- Fork the repository https://github.com/rhd-gitops-example/gitops-example-dev
+- Fork the repository https://github.com/rhd-gitops-example/gitops-example-staging
+- Inside the services folder build the code: `go build ./cmd/services`
 - export GITHUB_TOKEN=[your token]
 - Substitute your repository URLs for those in square brackets:
+- Add `--branch-name [unique branch name]` to the command below to create and push to the branch name (if not added will create a default branch)
```

Comment:
Now that https://github.com/rhd-gitops-example/services/pull/32 is merged we shouldn't have to use --branch-name very often.